.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,171 @@
+name: Bug Report
+description: File a bug report
+labels: ["A: Bug"]
+# labels: ["A: Bug", "A: By Design", "A: Cannot reproduce", "A: Closed", "A: In progress", "A: Invalid", "A: Problematic", "A: Rejected", "A: Resolved", "A: Won't fix", "Extra", "Extra (zmc)", "Infrastructure", "N: AdGuard Browser Extension", "N: AdGuard Content Blocker", "N: AdGuard DNS", "N: AdGuard for Android", "N: AdGuard for iOS", "N: AdGuard for Mac", "N: AdGuard for Safari", "N: AdGuard for Windows", "N: Other", "NSFW", "P1: Critical", "P2: High", "P3: Medium", "P4: Low", "T: Ads", "T: Annoyance", "T: Anti Adblock Script", "T: Incorrect Blocking", "T: Popups", "T: Social Widget", "T: Stealth Mode issue", "T: Tracker"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: |
+        To avoid invalid reports, please check and confirm following checkboxes.
+      options:
+        - label: This site DOES NOT contains sexually explicit material;
+          required: true
+        - label: DNS filters were updated before redroduced an issue;
+          required: false
+        - label: AdGuard product version is up-to-date;
```

Comment:
up-to-date points are also not necessary needed

Reply:
Even for apps? Or all ad blocker apps can be removed from this template?